modules/opensearch/includes/opensearch.search.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * Implements handling of loans through OpenRuth.
+ */
+
+/**
+ * Get a list of material types from the Well.
+ */
+function opensearch_search_material_types() {
```

Comment:
I am thinking whether it is a good idea to add a implementation of the `search` provider and the `material_types` call to [the dummy Connie provider](https://github.com/ding2/ding2/tree/master/modules/ding_provider/connie). This is the closest form of documentation we have of providers currently.

On the other hand it may cause problems if you want to combine OpenSearch with a dummy library system provider or Alma with a dummy search provider.